### DISCUSSION TITLE
TableHeaderCell Padding - How do I set this property/attribute?

### DISCUSSION THREAD

**Discussion by johnhbarrett** [Megabit/Blazorise#3909] [Q&A] [answered]
<TableHeaderCell Padding=""
  
  it's asking for an IFluentSpacing type but I can't work it out or find the doco

Any tips appreciated!

**Comment by stsrki** [comment, marked as answer]
`<TableHeaderCell Padding="Padding.Is3.FromTop.Is2.FromBottom"`

https://blazorise.com/docs/helpers/utilities
